docker-build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
We need to avoid using such scripts, because dockerfile is being used in CICD on hub.docker.com to build the image. And it knows nothing about SH scrips. So we should use multistage dockerfile to build and then use created war